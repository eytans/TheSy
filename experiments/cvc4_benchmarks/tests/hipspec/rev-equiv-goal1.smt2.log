19:39:06 [WARN] Collecting statistics
19:39:06 [WARN]  ex is: zero
19:39:06 [WARN]  ex is: (succ zero)
19:39:06 [WARN]  ex is: (succ (succ zero))
19:39:06 [WARN]  ex is: nil
19:39:06 [WARN]  ex is: (cons autovar_Nat_0 nil)
19:39:06 [WARN]  ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:39:06 [WARN] Running exploration without proof mode, but goals were given
19:39:06 [WARN] Proof mode: true
19:39:06 [WARN]  ex is: zero
19:39:06 [WARN]  ex is: (succ zero)
19:39:06 [WARN]  ex is: (succ (succ zero))
19:39:06 [WARN]  ex is: nil
19:39:06 [WARN]  ex is: (cons autovar_Nat_0 nil)
19:39:06 [WARN]  ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:39:06 [WARN] Running exploration without proof mode, but goals were given
19:39:06 [WARN] Proof mode: true
19:39:06 [WARN] Found all lemmas
