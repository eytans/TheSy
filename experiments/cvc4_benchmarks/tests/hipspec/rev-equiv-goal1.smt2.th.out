19:39:06 [0m[33m[WARN] [0mCollecting statistics
(append nil ?x) => ?x
(append (cons ?x0 ?y) ?z) => (cons ?x0 (append ?y ?z))
(cons ?x0 (append ?y ?z)) => (append (cons ?x0 ?y) ?z)
(rev nil) => nil
nil => (rev nil)
(rev (cons ?x1 ?y)) => (append (rev ?y) (cons ?x1 nil))
(append (rev ?y) (cons ?x1 nil)) => (rev (cons ?x1 ?y))
(qreva nil ?x) => ?x
(qreva (cons ?z2 ?x) ?y) => (qreva ?x (cons ?z2 ?y))
(qreva ?x (cons ?z2 ?y)) => (qreva (cons ?z2 ?x) ?y)
(qrev ?x) => (qreva ?x nil)
(qreva ?x nil) => (qrev ?x)
19:39:06 [0m[33m[WARN] [0m ex is: zero
19:39:06 [0m[33m[WARN] [0m ex is: (succ zero)
19:39:06 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:39:06 [0m[33m[WARN] [0m ex is: nil
19:39:06 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_0 nil)
19:39:06 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:39:06 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:39:06 [0m[33m[WARN] [0mProof mode: true
19:39:06 [0m[33m[WARN] [0m ex is: zero
19:39:06 [0m[33m[WARN] [0m ex is: (succ zero)
19:39:06 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:39:06 [0m[33m[WARN] [0m ex is: nil
19:39:06 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_0 nil)
19:39:06 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:39:06 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:39:06 [0m[33m[WARN] [0mProof mode: true
19:39:06 [0m[33m[WARN] [0mFound all lemmas
done in 1
